provider/cluster/kube/client.go

```diff
@@ -188,18 +191,19 @@ func (c *client) ServiceLogs(ctx context.Context, lid mtypes.LeaseID,
 	pods, err := c.kc.CoreV1().Pods(lidNS(lid)).List(ctx, metav1.ListOptions{})
 	if err != nil {
 		c.log.Error(err.Error())
-		return nil, errors.New("internal error")
+		return nil, errors.Wrap(err, ErrInternalError.Error())
 	}
 	streams := make([]*cluster.ServiceLog, len(pods.Items))
 	for i, pod := range pods.Items {
-		stream, err := c.kc.CoreV1().Pods(lidNS(lid)).GetLogs(pod.Name, &corev1.PodLogOptions{
+		var stream io.ReadCloser
+		stream, err = c.kc.CoreV1().Pods(lidNS(lid)).GetLogs(pod.Name, &corev1.PodLogOptions{
```

Comment:
why?

Reply:
nevermind, that was coming from errcheck dependency. rid off